kotest-framework/kotest-framework-engine/src/jvmTest/kotlin/com/sksamuel/kotest/engine/test/timeout/GlobalTimeoutTest.kt

```diff
@@ -1,40 +1,39 @@
 package com.sksamuel.kotest.engine.test.timeout
 
-import io.kotest.assertions.asClue
 import io.kotest.core.config.ProjectConfiguration
 import io.kotest.core.spec.style.FunSpec
 import io.kotest.core.spec.style.StringSpec
 import io.kotest.engine.TestEngineLauncher
 import io.kotest.engine.listener.CollectingTestEngineListener
-import io.kotest.matchers.collections.shouldContainExactly
 import io.kotest.matchers.shouldBe
 import kotlinx.coroutines.delay
-import kotlin.time.Duration.Companion.days
 
 class GlobalTimeoutTest : FunSpec() {
    init {
-      test("global timeouts should apply if no other timeout is set") {
-         val c = ProjectConfiguration().apply { timeout = 3 }
+      test("global timeouts should apply if no other timeout is set")
+      {
+         val c = ProjectConfiguration().apply { timeout = 2000 }
          val collector = CollectingTestEngineListener()
          TestEngineLauncher(collector)
             .withClasses(TestTimeouts::class)
             .withConfiguration(c)
             .launch()
-
-         collector.waitForEnginesFinished()
-
-         collector.names.shouldContainExactly("blocked", "suspend")
-
-         collector.result("blocked").asClue { result -> result?.isError shouldBe true }
-         collector.result("suspend").asClue { result -> result?.isError shouldBe true }
+         collector.tests.mapKeys { it.key.name.testName }["blocked"]?.isError shouldBe true
+         collector.tests.mapKeys { it.key.name.testName }["suspend"]?.isError shouldBe true
       }
```

Comment:
Can the assertions be kept? They're more useful than the ones in this PR.

Reply:
You mean these?
```kotlin
         collector.names.shouldContainExactly("blocked", "suspend")

         collector.result("blocked").asClue { result -> result?.isError shouldBe true }
         collector.result("suspend").asClue { result -> result?.isError shouldBe true }
```

No problem.